### DISCUSSION TITLE
Stork static list?

### DISCUSSION THREAD

**Discussion by edeandrea** [quarkusio/quarkus#25049] [Q&A] [answered]
Using SmallRye stork - is there a built-in service discovery mechanism for defining a static list of URLs?

I thought I saw that at one point, but [the SmallRye Stork documentation](https://smallrye.io/smallrye-stork) doesn't mention it at all.

[a comment by an automated account is not shown]

**Comment by michalszynkiewicz** [comment, marked as answer]
It got documented after the latest release. It will be available on the documentation site with the next release.

Here's the documentation: https://github.com/smallrye/smallrye-stork/blob/main/docs/service-discovery/static-list.md

**Reply by edeandrea** [reply]
Thanks! Thats what I was looking for!
